web/src/message_list.js

```diff
@@ -447,21 +447,22 @@ export class MessageList {
         if ($row.find(".message_edit_form form").length !== 0) {
             return;
         }
-        $row.find(".message_edit_form").append($form);
+        $row.find(".messagebox-content").append($form);
```

Comment:
Should we add a comment in `single_message.hbs` noting the place where the message edit form will be dynamically added?

Reply:
Surely! Should I comment it at the top of the file or just below the `{{> message_body}}` partial?

Reply:
Just below the partial seems right to me, since that's where we're doing the surgery.